### PR TITLE
Initialise chat_bot

### PR DESCRIPTION
**Pull request by raideva**

### CODE REVIEW

File: codefest/settings.py [hunk 1]
```diff
@@ -220,4 +222,16 @@
...
+
+
+ASGI_APPLICATION = 'codefest.asgi.application'
+CHANNEL_LAYERS = {
+    'default': {
+        # 'BACKEND': 'channels_redis.core.RedisChannelLayer',
+        # 'CONFIG': {
+        #     "hosts": [('127.0.0.1', 6379)],
+        # },
+        'BACKEND': 'channels.layers.InMemoryChannelLayer',
```
Comment (nishantwrp) [inline codefest/settings.py]: Will this work when we deploy on heroku dynos @krashish8?

Comment (krashish8) [reply codefest/settings.py]: Not sure about it, but it seems that this might require some additional configurations, maybe in the Procfile?

Comment (nishantwrp) [reply codefest/settings.py]: I mean the dyno restarts every 5 mins. So, maybe the RAM is cleared too. I think we can use redis (Heroku has a add-on for that).

Comment (krashish8) [reply codefest/settings.py]: Yes, I agree.

I think the dyno restarts every 30 mins, but we have Uptime Robot to counter that (just a workaround). Well, the Uptime Robot is still active, and Amri bhaiya's dyno hours are consumed every month 🤭

File: codefest/settings.py [hunk 2]
```diff
@@ -220,4 +222,16 @@
...
+    CELERY_BROKER_URL=config('CLOUDAMQP_URL')
+
+
+ASGI_APPLICATION = 'codefest.asgi.application'
+CHANNEL_LAYERS = {
+    'default': {
+        # 'BACKEND': 'channels_redis.core.RedisChannelLayer',
+        # 'CONFIG': {
+        #     "hosts": [('127.0.0.1', 6379)],
+        # },
```
Comment (shubhanshu02) [inline codefest/settings.py:227]: To be enabled while deploying, right? Just wanted to confirm if the `channels.layers.InMemoryChannelLayer` is temporary for development purposes.

File: chat_bot/models.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from django.db import models
+
+# Create your models here.
+class Question(models.Model):
+    question = models.CharField(max_length=1024)
```
Comment (shubhanshu02) [inline chat_bot/models.py:5]: Can we use a `TextField` instead since max_length is large?

File: codefest/settings.py (hunk 1 above)
Comment (shubhanshu02) [reply codefest/settings.py]: I guess this part is only for development purposes. :thinking: It was written in the docs that:
> Channels also comes packaged with an in-memory Channels Layer. This layer can be helpful in for Testing or local-development purposes:

File: codefest/settings.py (hunk 2 above)
Comment (raideva) [reply codefest/settings.py:227]: Yes for deploying we have to add channels-reddis also in the requirement.txt, as it is currently not present there.

File: chat_bot/models.py (hunk 3 above)
Comment (raideva) [reply chat_bot/models.py:5]: Should i decrease the max length?

Comment (shubhanshu02) [reply chat_bot/models.py:5]: If the user types a paragraph-like message, we wouldn't be able to store that. Using a `models.TextField` will easily store that.

**Review by shubhanshu02 — APPROVED** [review]
Seems fine to me. Thanks @raideva :smile:

**Pull request merged**